core/config/docs/chains-evm.toml

```
@@ -37,6 +37,9 @@ ChainType = 'arbitrum' # Example
 # A re-org occurs at height 46 starting at block 41, transaction is marked for rebroadcast
 # A re-org occurs at height 47 starting at block 41, transaction is NOT marked for rebroadcast
 FinalityDepth = 50 # Default
+# FinalityTag means that the chain supports the finalized block tag when querying for a block. If FinalityTag is set to true for a chain, then FinalityDepth field is ignored.
+# Finality for a block is solely defined by the finality related tags provided by the chain's RPC API. This is a placeholder and hasn't been implemented yet.
+FinalityTag = false # Default
```

Would it make sense to suffix this with `*Enabled`?
```suggestion
FinalityTagEnabled = false # Default
```
Or maybe prefix `Use*`?
```suggestion
UseFinalityTag = false # Default
```

---

Unless it is a common pattern in Golang, I'd prefer not to change it. It's pretty self-explanatory.

---

I actually find it pretty confusing currently. I would have guessed that it served to override the actual tag value in RPC or something.  :shrug: 
I don't think this a Go question. These just serve the TOML, so the existing standards from other fields seem more relevant. There is precedent for both `Use*` and `*Enabled`.

---

If it's not clear then we should change it. I prefer `*Enabled` for consistency since we already have `ForwardersEnabled`.